CHANGELOG.rst

```diff
@@ -1,3 +1,12 @@
+0.7.2 (develop)
```

Comment:
Isn't this the version you are deploying?

Reply:
Yes, I messed up the tags earlier...so I will update that.